src/main/java/PalindromeChecker.java

```diff
@@ -3,11 +3,14 @@
 
     public boolean check(String word) {
 
+        //still needed?
         if(word.length() == 1) {
             return true;
         }
 
+        //reassigning? - just question-marks?
         word = word.toLowerCase().replaceAll(" ", "").replaceAll("\\?", "");
```

Comment:
Your regex just handles spaces and question-marks if I'm reading it right?  I suspect you might want to cover a wider set of characters (such as all punctuation marks) and there a bunch of useful character classes already defined in the Pattern class - see the Javadoc https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html